katello/hooks/pre/10-reset_feature.rb

```diff
@@ -135,6 +136,11 @@ def reset_pulp
   )
 end
 
+def reset_pulpcore
+  Kafo::KafoConfigure.logger.info 'Dropping Pulpcore database!'
+  execute('runuser - postgres -c "dropdb pulpcore"') if pulpcore_enabled?
```

Comment:
Let's move the `if` to the top of the function as a guard. No use logging we are dropping the pulpcore database if we actually aren't.

Reply:
Also, should this function look more like https://github.com/theforeman/foreman-installer/pull/459/files#diff-ed86b943421faaceae73f8a3bff3e2edR59 since that uses code already present and covers remote database?

Reply:
I raised an issue the other day around remote dbs..I am not sure that is supported yet..
https://projects.theforeman.org/issues/28901

Reply:
It will be, so I would opt for supporting it to not have to come re-engineer this code. @wbclark is working on the remote support

Reply:
Updated with the remote db part and running reset_pulpcore only if pulpcore_enabled? is true.